src/api/admin.py

```diff
@@ -230,19 +230,20 @@ async def update_alcohol(
     await AlcoholFilterDatabaseHandler.update_filters(
         db.alcohol_filters, db_alcohol['kind'], db_alcohol['type'], db_alcohol['country'], db_alcohol['color']
     )
-    new_image_name = payload.name.lower().replace(' ', '_')
-    old_image_name = old_alcohol['name'].lower().replace(' ', '_')
-    if payload.name and (new_image_name != old_image_name):
-        cloudinary.uploader.rename(
-            f'{settings.ALCOHOL_IMAGES_DIR}/{old_image_name}_sm',
-            f'{settings.ALCOHOL_IMAGES_DIR}/{new_image_name}_sm',
-            invalidate=True
-        )
-        cloudinary.uploader.rename(
-            f'{settings.ALCOHOL_IMAGES_DIR}/{old_image_name}_md',
-            f'{settings.ALCOHOL_IMAGES_DIR}/{new_image_name}_md',
-            invalidate=True
-        )
+    if payload.name:
+        new_image_name = payload.name.lower().replace(' ', '_')
+        old_image_name = old_alcohol['name'].lower().replace(' ', '_')
+        if new_image_name != old_image_name:
```

Comment:
przerzuć to do ifa wyżej
payload.name != old_alcohol.name powinno wystarczyć

Reply:
Jeśli tak bym zrobił to weszłoby do ifa kiedy payload.name jest puste i wtedy sypnie się na payload.name.lower :c
Może teraz taka sytuacja się nie zdarzy ale w sumie nie wiemy czy kiedyś nie zmieni czegoś daro

Reply:
`if payload.name and payload.name != old_alcohol.name` Przecież jak tu nie będzie payload.name to część po AND się nie wykona

Reply:
dobra, zrozumiałem że miałoby zostać samo payload.name != old_alcohol.name XD